CHANGELOG.md

```diff
@@ -42,17 +42,37 @@ This section is for maintaining a changelog for all breaking changes for the cli
 
 ## [Unreleased 2.x]
 
+### Added
+
+### Dependencies
+
+### Changed
+- Restore support for Java 8 ([#767](https://github.com/opensearch-project/opensearch-java/pull/767))
+- Add an integration test that runs on JDK-8 ([#795](https://github.com/opensearch-project/opensearch-java/pull/795))
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
```

Comment:
Line 36 and Line 39 also belong here and not in the Unreleased 3.0 section. Can we make that change?

Also, PR from line 36 is not backported to 2.x, I will do that backport.

Reply:
> Also, PR from line 36 is not backported to 2.x, I will do that backport.

@VachaShah I moved it to [Unreleased 2.x] since 2.8.2 is already shipped

Reply:
Thank you @reta!